plugins/polio/export_utils.py

```diff
@@ -0,0 +1,114 @@
+from ctypes import alignment
+from curses import start_color
+from turtle import color
+from openpyxl import Workbook
+from openpyxl.writer.excel import save_virtual_workbook
+from openpyxl.utils import get_column_letter
+from openpyxl.styles import Alignment, Font, PatternFill
+from openpyxl.styles.borders import Border, Side
+import datetime as dt
+import calendar
+
+# from datetime import datetime
+
+
+def generate_xlsx(filename, datas):
+    file = Workbook()
+    sheet = file.active
+    sheet.title = filename
+    columns = get_columns_names()
+    # display columns in the xlsx file
+    for row in range(1, 2):
+        for column in range(1, len(columns) + 1):
+            cell_header = sheet.cell(column=column, row=row, value=columns[column - 1])
+            cell_header = format_cell(cell_header, "12", True)
+            sheet = cell_dimension_pattern_fill(sheet, cell_header, None, 25.75, True)
```

Comment:
To separate the logic from the configuration and make things more readable, I would move hardcoded values such as 25.75 to some constant (CELL_WIDTH or something like that)